### PR TITLE
Fixes #203: Removed dist files.

### LINKED ISSUE #203: Don't version dist files, publish & host them.
Building and versioning dist files is a real pain; we should rather build and host them for each release on github pages along the demo I think. Related to #202.

### PR DESCRIPTION
**Pull request by n1k0**
Refs #203. Current demo at http://kinto.github.io/kinto.js uses the new public asset links.

r=? @leplatrem

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
       </li>
     </template>
   </div>
-  <script src="kinto.dev.js"></script>
+  <script src="http://mozilla-services.github.io/kinto.js/kinto.dev.js"></script>
```
Comment (Natim) [inline demo/index.html]: Shall we use https for that?

**Comment by Natim** [comment]
It looks good to me Also, I am in favour of the http/https switch.

File: demo/index.html (hunk 1 above)
Comment (n1k0) [inline demo/index.html]: > Shall we use https for that?

Good point; even better, using `//` for automatic current protocol conservation :)

**Comment by n1k0** [comment]
Hmm for some reason the hosted demo doesn't seem to update with latest changes http://mozilla-services.github.io/kinto.js/; I'm holding down merging for now.

**Comment by n1k0** [comment]
gh-pages branch has been properly updated, but hosted demo doesn't reflect it oO https://github.com/Kinto/kinto.js/blob/gh-pages/index.html

Cache issue? Delayed propagation?

**Comment by Natim** [comment]
Yes it could be the cache. One way to make sure it is, is to load the js file url and force refresh it.

**Comment by n1k0** [comment]
> One way to make sure it is, is to load the js file url

The problem isn't with the js file url but with the index.html one… purging browser cache doesn't fix it, and curl shows old version at its url. Weird.

**Comment by n1k0** [comment]
Woookay I'm just realizing we've been using a wrong demo URL since we've moved the project under the Kinto org. The correct URL is http://kinto.github.io/kinto.js/. Gonna update all the links to it.

**Comment by n1k0** [comment]
Requesting further validation with latest change; r=? @Natim @leplatrem

**Comment by leplatrem** [comment]
Gave a quick look at everything and it looks sane to me

**Comment by Natim** [comment]
r+

**Pull request merged**